airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -443,7 +449,7 @@ def parent_stream(self, stream):
 
     @property
     def cursor_field(self) -> Union[str, List[str]]:
-        return [self._cursor_field]
+        return self._cursor_field
```

Comment:
Why this change?

Reply:
Because we're using `current_state[self.cursor_field]` in the code below